src/components/UI/VersionPicker/VersionPicker.tsx

```diff
@@ -216,9 +238,9 @@ const VersionPicker = ({ onChange, selectedVersion, versions, ...props }) => {
 };
 
 VersionPicker.propTypes = {
+  selectedVersion: PropTypes.string.isRequired,
+  versions: PropTypes.array.isRequired,
   onChange: PropTypes.func,
```

Comment:
We can get rid of that one, right?  Or maybe we want to check this at runtime as well?

Reply:
You mean getting rid of proptypes completely? I wanted to discuss that 😬

Reply:
👍  I  meant getting rid only of `onChange` prop since we are checking it in `IVersionPickerProps` interface

Reply:
We'll get linter errors if we don't include all the props in the `propTypes` 😅